### DISCUSSION TITLE
深入浅出Nodejs读书笔记 - Tw93

### DISCUSSION THREAD

**Discussion by tw93** [tw93/tw93.github.io#98] [General]
http://tw93.com/2015-03-01/shen-ru-qian-chu-nodejs-reading-mind-map.html 

 这里是Tw93的技术博客，用于我的学习总结和研究，主要方向是前端，热爱分享、开源，最近在使用react和weex。Node今天终于把朴灵老师写的《深入浅出Node.js》给学习完了, 这本书不是一本简单的Node入门书籍，它没有停留在Node介绍或者框架、库的使用层面上，而是从不同的视角来揭示Node自己内在的特点和结构。建议有一定Node基础或者做过Node方面的小项目的同学阅读，看完以后你的思维会有很奇特的碰撞，我看的时候就常常会有这样的想法:“哦，原来这个功能是这样实现的哦”。下面这篇文章是我第二次阅读《深入浅出Node.js》的一些学习记录，并且通过百度脑图这个工具来画出思维导图，每天将自己的学习总结写在这篇文章下面 ...

**Comment by tw93** [comment]
评论替换

**Comment by hufans** [comment]
谢谢。

**Comment by ght5935** [comment]
脑图看不到了啊

**Comment by tw93** [comment]
> @ght5935
> 脑图看不到了啊
> 

我去更新一下
